domains/Discussions/Database/Migrations/2020_10_12_104113_create_questions_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Domains\Accounts\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateQuestionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('questions', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(User::class, 'author_id');
```

Comment:
I believe creating foreign keys doesn't implicitly create an index. We need to create one explicitly.

Reply:
It does in MySQL but not in others, so yes, let's explicitly add it.